internal/cloud/gcp/application_identity.go

```diff
@@ -76,6 +78,8 @@ func CreateApplicationIdentity(ctx context.Context, config *ApplicationIdentityC
 
 	config.ID = serviceAccountOutput.Email
 	config.ServiceAccountEmail = serviceAccountOutput.Email
+	// TODO: wire up k8s service account role grant after gcp is merged
+	// k8sEmail := fmt.Sprintf("%s.svc.id.goog[%s/%s]", config.Project, config.KubernetesNamespace, config.KubernetesServiceAccountName)
```

Comment:
Should this be added back in now that you’ll have the deets?

Reply:
Yeah I have it on my branch. I should be able to get my branch tested and solid tonight, including this k8s bit.